### DISCUSSION TITLE
You have codespaces that will soon be deleted

### DISCUSSION THREAD

**Discussion by faisaldarbar** [community/community#40778] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I received this message today that "You have codespaces that will soon be deleted". My concern is whether my code and repository will also be deleted when this happens?

**Comment by mohamedzhioua** [comment, marked as answer]
inactive codespaces will be deleted if they have been unused after a period , your code and repo will be fine

**Reply by faisaldarbar** [reply]
@mohamedzhioua thank you so much.. this was driving me crazy 😄

**Reply by ghost** [reply]
🙂

**Reply by mohamedzhioua** [reply]
the most important you are fine bro ^_^

**Reply by faisaldarbar** [reply]
@mohamedzhioua one more question.. how do I know whether I own the repository? Since it is mentioned "If a repository belongs to an organization, the organization admin may have set a retention period for the whole organization." Actually it is my CS50 course repository, so I may probably not own it!?

**Reply by mohamedzhioua** [reply]
@faisaldarbar to answer your question Just go to the team members tab within your repo, on the right it will describe what type of member they are. Ex: member or owner.
if you not the owner , you can ask to be a owner in it , and if you just want the repo and code , u can fork it

**Reply by faisaldarbar** [reply]
Ok.. thanks will do that.. thanks again 👍

**Comment by mohamedzhioua** [comment]
@faisaldarbar you can take a look here and you will see that only the code space is what going to be deleted , dont worry about your repo and code 
"https://docs.github.com/en/codespaces/customizing-your-codespace/configuring-automatic-deletion-of-your-codespaces"

**Reply by faisaldarbar** [reply]
@mohamedzhioua thanks a lot
